spec/takeaway_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  # subject(:takeaway) {described_class.new}
+  # let(:menu) {double(:menu)}
+  # let(:order) {double(:order)}
+
+  # it 'should display the menu' do
+  #   allow(menu).to receive(:options).and_return({'pizza' => 5.99, 'burger' => 4.99})
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)